content/applications/finance/payment_acquirers/razorpay.rst

```diff
@@ -36,10 +36,14 @@ Configuration on Odoo
 #. Configure the rest of the options to your liking.
 
 .. important::
-  - If you configure Odoo to capture amounts manually, make sure to set the
-    :guilabel:`Manual Capture` to :guilabel:`manual` on Razorpay. Otherwise, the transaction
-    would be blocked in the authorized state in Odoo. To do so, go to your Razorpay Dashboard and
-    then to :menuselection:`Settings --> Configuration`.
+  If you configure Odoo to capture amounts manually:
+
+  - Make sure to set the :guilabel:`Manual Capture` to :guilabel:`manual` on Razorpay. Otherwise,
+    the transaction would be blocked in the authorized state in Odoo. To do so, go to your Razorpay
```

Comment:
```suggestion
    the transaction would be blocked at the authorized state in Odoo. To do so, go to your Razorpay
```
I'm not sure I fully understand the flow just by reading, so this is just a suggestion. If the transaction is stopped at the step of authorizing, then I would use "at". :)

Reply:
It's the status/state of the transaction, more than a step, so IMHO, "in" might be more adapted?

Reply:
I mean, the transaction will be processed and done, but on Odoo side, the transaction will be kept in the wrong state (maybe this is clearer? :thinking: )

Reply:
> I mean, the transaction will be processed and done for Razorpay, but on Odoo side, the transaction will be kept in the wrong state (maybe this is clearer? thinking ), so no payment created etc

I think this might be clearer for users, yes :)

Reply:
Checking right now, this was a copy/paste from Adyen and not needed anyway for Razorpay, so I took it off